pkg/daemon/ceph/osd/agent.go

```diff
@@ -26,13 +26,15 @@ import (
 	"strings"
 	"time"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
 	"github.com/google/uuid"
 
 	rookalpha "github.com/rook/rook/pkg/apis/rook.io/v1alpha1"
 	"github.com/rook/rook/pkg/clusterd"
 	"github.com/rook/rook/pkg/daemon/ceph/mon"
+	"github.com/rook/rook/pkg/operator/k8sutil"
 	"github.com/rook/rook/pkg/util"
-	"github.com/rook/rook/pkg/util/kvstore"
```

Comment:
as discussed, let's look into if there was a need to move the kvstore package.  I'm hoping that will reduce the size of this diff.

Reply:
since the removal of standalone, it looked like we didn't need the kvstore interface since k8s was the only implementation now. This should have been a separate PR though. I'll look at splitting it out